packages/eas-cli/src/build/utils/updates.ts

```diff
@@ -0,0 +1,14 @@
+import { ExpoConfig, getPackageJson } from '@expo/config';
```

Comment:
Maybe this file should be placed outside the `src/build` directory?

Reply:
if we will need this code outside that would also mean that configuration should also be extracted from the build directory